scripts/init.sh

```diff
@@ -9,3 +9,6 @@ cp files/init/start.sh /etc/hobby-hub/
 # Program manager
 mkdir -p /etc/hobby-hub/out/ # contains output of program manager
 cp files/src/program_manager.py /etc/hobby-hub/ # program manager itself
+
+# Pin mapping configuration
+echo "{}" | sudo tee -a /etc/hobby-hub/pin_mapping.json # creates empty, but valid JSON file
```

Comment:
You've created the json file here but haven't copied the pin_manager.py . This is fine assuming you plan to add this to the python module. in which case we'll have to figure out where to place this later...